fdbserver/RestoreLoader.actor.cpp

```diff
@@ -138,15 +154,33 @@ void handleRestoreSysInfoRequest(const RestoreSysInfoRequest& req, Reference<Res
 	}
 
 	self->appliersInterf = req.sysInfo.appliers;
+	// Update rangeVersions
+	ASSERT(self->rangeVersions.size() == 1); // rangeVersions has not been set
```

Comment:
Nit. Add assertion for `req.rangeVersions` size is > 0.

Reply:
This assertion is to double check that the `RestoreSysInfoRequest` is not processed more than once. The operation on `rangeVersions ` is not idempotent .  I'm adding the extra assertion as well.